client/package.json

```diff
@@ -2,9 +2,12 @@
   "private": true,
   "scripts": {
     "dev": "next",
+    "test": "jest --coverage",
+    "lint": "eslint --ext .js,.ts .",
+    "lint:fix": "npm run lint -- --fix",
     "build": "next build",
     "start": "next start",
-    "typecheck": "tsc"
+    "typecheck": "tsc --watch"
```

Comment:
create a new entry for watch

"typecheck:watch"

Reply:
this typecheck is just for call ts package ?

Reply:
I don't know what this `tsc` does, But if you want to add this parameter, then create a new entry

Reply:
i saw that it returns the typescript version i guess

Reply:
but no make sense you add `--watch` in `typecheck`

Reply:
`npx tsc.`

"You can include the above command in your scripts to make it easier to run. Go to package.json and add the` --watch` flag to watch the files for changes. "

i think this is to update the changes of the page's build,
i ran "typecheck" and it returns the build info.

Reply:
i removed it